src/helm/benchmark/presentation/create_plots.py

```diff
@@ -0,0 +1,583 @@
+import argparse
+from collections import defaultdict
+from dataclasses import dataclass
+from datetime import date
+import json
+import os
+from typing import List, Dict, Optional, Any, Callable, Union, Mapping, Tuple, Set
+
+import colorcet
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.stats import pearsonr
+import seaborn as sns
+
+from helm.benchmark.presentation.schema import read_schema
+
+sns.set_style("whitegrid")
+
+DOWN_ARROW = "\u2193"
+UP_ARROW = "\u2191"
+metric_group_to_label = {
+    "Accuracy": f"Accuracy {UP_ARROW}",
+    "Calibration": f"Calibration error {DOWN_ARROW}",
+    "Robustness": f"Robustness {UP_ARROW}",
+    "Fairness": f"Fairness {UP_ARROW}",
+    "Bias": f"Bias (gender repr.) {DOWN_ARROW}",
+    "Toxicity": f"Toxicity {DOWN_ARROW}",
+    "Efficiency": f"Inference time (s) {DOWN_ARROW}",
+}
+
+
+@dataclass
+class Column:
+    """Values and metadata for each column of the table."""
+
+    name: str
+    group: str
+    metric: str
+    values: np.ndarray
+    lower_is_better: Optional[bool]
+
+
+@dataclass
+class Table:
+    """Column-based representation of a standard run-group table. See summarize.py for exact documentation."""
+
+    adapters: List[str]
+    columns: List[Column]
+    mean_win_rates: Optional[np.ndarray] = None
+
+
+def parse_table(raw_table: dict) -> Table:
+    """Convert raw table dict to a Table. Ignores strongly contaminated table entries."""
+
+    def get_cell_values(cells: List[dict]) -> List[Any]:
+        values = []
+        for cell in cells:
+            value = cell["value"] if "value" in cell else np.nan
+            if "contamination_level" in cell and cell["contamination_level"] == "strong":
+                value = np.nan
+            values.append(value)
+        return values
+
+    adapters: Optional[List[str]] = None
+    columns: List[Column] = []
+    mean_win_rates: Optional[np.ndarray] = None
+    for header_cell, *column_cells in zip(raw_table["header"], *raw_table["rows"]):
+        cell_values = get_cell_values(column_cells)
+        if header_cell["value"] == "Model/adapter":
```

Comment:
This is a bit brittle...could we define this as a constant in `summarize.py` and use that same constant here?